chase/frontend/app/participant/layout.tsx

```diff
@@ -6,13 +6,9 @@ export default function Participant_Pages_Layout({
   children: React.ReactNode;
 }) {
   return (
-    <div className="flex h-screen w-screen bg-gray-light">
-      <div className="m-3 rounded-3xl bg-white flex shadow-md w-full overflow-hidden">
-        <div className="flex-1 flex">
-          <Navbar />
-          {children}
-        </div>
-      </div>
+    <div className="flex h-screen w-screen bg-white shadow-md overflow-hidden">
```

Comment:
I decided to get rid of the aesthetic border now, since I had a few challenges that made it easier for me to accept your request to omit it in the first place ;)